pkg/azure/blob/credentials.go

```diff
@@ -1,11 +1,13 @@
-package credentials
+package blob
 
 import (
 	"os"
 	"regexp"
 
+	"get.porter.sh/plugin/azure/pkg/azure/config"
```

Comment:
I wrote this originally in Oct and in hindsight I hate that I named this package config too. It makes it hard to tell if I'm using porter's config or this one. Do you have suggestions for differentiating the package name and/or the Config struct name?

Reply:
Perhaps just name the import as `azureconfig` or some such when used?  And/or `AzureConfig` as the struct?